pipelines/PacBio/VariantCalling/PBCCSWholeGenome.wdl

```diff
@@ -14,6 +8,31 @@ import "../../../tasks/Utility/Finalize.wdl" as FF
 import "../../../tasks/QC/SampleLevelAlignedMetrics.wdl" as COV
 
 workflow PBCCSWholeGenome {
+
+    meta {
+        description: "A workflow that performs single sample variant calling on PacBio HiFi reads from one or more flow cells. The workflow merges multiple samples into a single BAM prior to variant calling."
```

Comment:
Again, not multiple samples but multiple SMRT cells.

Reply:
done